packages/graphback-core/src/plugin/GraphbackCoreMetadata.ts

```diff
@@ -57,10 +58,14 @@ export class GraphbackCoreMetadata {
     return this.resolvers;
   }
 
+  public getModelDefinitions() {
+    return this.models;
+  }
+
   /**
    * Get Graphback Models - GraphQL Types with additional CRUD configuration
    */
-  public getModelDefinitions() {
+  public setModelDefinitions() {
```

Comment:
I'd expect a setter method to receive models as parameters but here we are not doing that but instead we are parsing/computing model definitions. can we rename the method in that regard?

Reply:
I was not sure what to call it, because it is still technically setting `this.models`. I've renamed it to `buildAndSetModelDefinitions`. Is that suitable?